frontend/src/metabase/components/SegmentedControl.styled.js

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+import { color } from "metabase/lib/colors";
+
+const BORDER_RADIUS = "8px";
+
+export const SegmentedList = styled.ul`
+  display: flex;
+`;
+
+export const SegmentedItem = styled.li`
+  display: flex;
+  align-items: center;
+  font-weight: bold;
+  cursor: pointer;
+  color: ${props => (props.isSelected ? color(props.selectedColor) : null)};
+  padding: 6px 12px;
+
+  border: 1px solid ${color("border")};
+  border-right-width: ${props => (props.isLast ? "1px" : 0)};
+  border-top-left-radius: ${props => (props.isFirst ? BORDER_RADIUS : 0)};
+  border-bottom-left-radius: ${props => (props.isFirst ? BORDER_RADIUS : 0)};
+  border-top-right-radius: ${props => (props.isLast ? BORDER_RADIUS : 0)};
+  border-bottom-right-radius: ${props => (props.isLast ? BORDER_RADIUS : 0)};
+
+  aria-selected: ${props => props.isSelected};
```

Comment:
Not sure if CSS understands `aria` attributes 🤔 
Also it seems like you are using native radio buttons so do we really need to add `aria` attributes? I think voiceover should be working correctly without it, correct me if I'm wrong

Reply:
Ouch, you're right, I just copied that from `Radio` 😄 Will remove it from there as well